app/controllers/application_controller.rb

```diff
@@ -4,6 +4,13 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
 
   def  aaa
-      1 + 1
+  1 + 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1+ 1 + 1
```

Comment:
Line is too long. [165/80]<br>Use 2 (not 0) spaces for indentation.<br>Surrounding space missing for operator `+`.